packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/aws-custom-resource/integ.aws-custom-resource-athena.ts

```diff
@@ -134,9 +134,20 @@ const notebook = new AwsCustomResource(stack, 'AthenaNotebook', {
 });
 notebook.node.addDependency(workgroup);
 
-new IntegTest(app, 'CustomResourceAthena', {
+const integTest = new IntegTest(app, 'CustomResourceAthena', {
   testCases: [stack],
   diffAssets: true,
 });
 
-app.synth();
+// Assertion to check if TestWG workgroup is present in the list of workgroups
+integTest.assertions.awsApiCall('Athena', 'listWorkGroups')
+  .expect(ExpectedResult.objectLike({
+    WorkGroups: [
+      {
+        Name: 'TestWG',
+      },
+      {
+        Name: 'primary',
```

Comment:
Any idea where `primary` is coming from? It looks like this integ test is just creating a work group named `TestWG`. Let's format like this:
```ts
WorkGroups: Match.arrayLike([
	{
		Name: workgroupName,
	},
]),
```

Reply:
The `primary` workgroup got auto created while running this integ test and I'm not able to do a clean up on it as well. So, was of the assumption that it would initially get created but, can check on the name. Also, will use the variable name instead of hardcoding it.

Reply:
You may be right, I haven't looked into it enough. I think we should still omit it since we just want to assert against the creation of the specific work group. No need to any cleanup. Let's just use `arrayLike`. This will be safer since we don't necessarily know if `primary` is always created, but we do know that `TestWG` will be created.